vnext/src/Libraries/Pressability/Pressability.windows.js

```diff
@@ -426,6 +426,7 @@ export default class Pressability {
   |}>;
   _touchActivateTime: ?number;
   _touchState: TouchState = 'NOT_RESPONDER';
+  _isPressed : boolean = false;
```

Comment:
Nit: can we change this to something more specific to key events so this state doesn't get confused by someone in the future for something else? `pressed` is quite overloaded in this module.

Perhaps `_isKeyDown`?